From 0db4828b71aae18b00804c0755988ac2874ba17e Mon Sep 17 00:00:00 2001
From: Eric Kinzie <eric@qosient.com>
Date: Wed, 30 May 2018 00:20:17 -0400
Subject: [PATCH] netconf/perl: remove requirement for perl v5.018

RHEL 7 ships with perl v5.016 and there does not appear to be any hard
requirement for a newer version.  Remove the "use" statements for a
particular version of perl.
---
 netconf/perl/yangcli/Makefile.PL    | 1 -
 netconf/perl/yangcli/lib/yangcli.pm | 2 --
 netconf/perl/yangrpc/Makefile.PL    | 1 -
 netconf/perl/yangrpc/lib/yangrpc.pm | 2 --
 netconf/perl/yuma/Makefile.PL       | 1 -
 netconf/perl/yuma/lib/yuma.pm       | 2 --
 6 files changed, 9 deletions(-)

diff --git a/netconf/perl/yangcli/Makefile.PL b/netconf/perl/yangcli/Makefile.PL
index ab0eb0b..6df1f47 100644
--- a/netconf/perl/yangcli/Makefile.PL
+++ b/netconf/perl/yangcli/Makefile.PL
@@ -1,4 +1,3 @@
-use 5.018002;
 use ExtUtils::MakeMaker;
 # See lib/ExtUtils/MakeMaker.pm for details of how to influence
 # the contents of the Makefile that is written.
diff --git a/netconf/perl/yangcli/lib/yangcli.pm b/netconf/perl/yangcli/lib/yangcli.pm
index 98169ac..5a6bd26 100644
--- a/netconf/perl/yangcli/lib/yangcli.pm
+++ b/netconf/perl/yangcli/lib/yangcli.pm
@@ -1,7 +1,5 @@
 package yangcli;
 
-#use 5.020002;
-use 5.018002;
 use strict;
 use warnings;
 
diff --git a/netconf/perl/yangrpc/Makefile.PL b/netconf/perl/yangrpc/Makefile.PL
index 137a66d..c09241b 100644
--- a/netconf/perl/yangrpc/Makefile.PL
+++ b/netconf/perl/yangrpc/Makefile.PL
@@ -1,4 +1,3 @@
-use 5.018002;
 use ExtUtils::MakeMaker;
 # See lib/ExtUtils/MakeMaker.pm for details of how to influence
 # the contents of the Makefile that is written.
diff --git a/netconf/perl/yangrpc/lib/yangrpc.pm b/netconf/perl/yangrpc/lib/yangrpc.pm
index 42680b8..a22cd36 100644
--- a/netconf/perl/yangrpc/lib/yangrpc.pm
+++ b/netconf/perl/yangrpc/lib/yangrpc.pm
@@ -1,7 +1,5 @@
 package yangrpc;
 
-#use 5.020002;
-use 5.018002;
 use strict;
 use warnings;
 
diff --git a/netconf/perl/yuma/Makefile.PL b/netconf/perl/yuma/Makefile.PL
index 21f8203..e60ed24 100644
--- a/netconf/perl/yuma/Makefile.PL
+++ b/netconf/perl/yuma/Makefile.PL
@@ -1,4 +1,3 @@
-use 5.018002;
 use ExtUtils::MakeMaker;
 # See lib/ExtUtils/MakeMaker.pm for details of how to influence
 # the contents of the Makefile that is written.
diff --git a/netconf/perl/yuma/lib/yuma.pm b/netconf/perl/yuma/lib/yuma.pm
index 0846f66..296562b 100644
--- a/netconf/perl/yuma/lib/yuma.pm
+++ b/netconf/perl/yuma/lib/yuma.pm
@@ -1,7 +1,5 @@
 package yuma;
 
-#use 5.020002;
-use 5.018002;
 use strict;
 use warnings;
 
-- 
1.8.3.1

